python/cudf/cudf/core/multiindex.py

```diff
@@ -241,6 +241,81 @@ def labels(self):
         )
         return self.codes
 
+    def isin(self, values, level=None):
+        """Return a boolean array where the index values are in values.
+
+        Compute boolean array of whether each index value is found in
+        the passed set of values. The length of the returned boolean
+        array matches the length of the index.
+
+        Parameters
+        ----------
+        values : set, list-like, Index or Multi-Index
```

Comment:
If this is an `Index` we throw, no?

Reply:
`Index` is allowed if `level` is provided